README.md

```diff
@@ -27,3 +27,6 @@ Our Developer Success Team can assist with answering any questions as well as pr
   http://www.apache.org/licenses/
   Full details available within the LICENSE file.
 ```
+
+## 4. Reflecting Re-architecting API changes in the documentation website
+If the API ever has a restructuring, there will be a need to update the playfab spec: https://www.playfabapi.com/apispec/ to reflect changes in the doc site.
```

Comment:
We actually have to update the TOC.json to match the results from that url.
It's not actually possible to change the results of that url from anything but pf-main code.

Reply:
If that's the case, should we move this mention over to that repo's README.md instead?